src/pages/index.tsx

```diff
@@ -21,15 +22,20 @@ export default function Home() {
   const router = useRouter()
   const [copied, setCopied] = useState(false);
 
-const HandleOnCopy = useCallback(() => {
+const handleOnCopy = useCallback((title: string) => {
     try {
       setCopied(true);
       setTimeout(() => setCopied(false), 2000);
+      copyToClipboardAction(title)
     } catch (error) {
       //handle errors
     }
   }, [setCopied]);
 
+  const onRPCScroll = useCallback(() => {
+    scrollToAction('Get RPC Details')
+  }, [])
```

Comment:
why a useCallback here and not just declare it as a static new function (outside of the rendering of the component?